conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
Switched to cross-compilation as Travis CI `linux_ppc64le` CI was dying before the package build even started. Appears this helped resolve that issue